src/devtools/views/actionRegion.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import withSend from 'microcosm/addons/with-send'
+
+export default withSend(
+  class ActionRegion extends React.Component {
```

Comment:
@efatsi We ended up abstracting that withSend out to a wrapper component so we can reuse it wherever we want instead of having a specialized ItemWrapper component just to send that action.